lib/engine/round/operating.rb

```diff
@@ -65,6 +80,16 @@ def description
         self.class::STEP_DESCRIPTION[@step]
       end
 
+      def pass_description
+        # If the user is continuing an action change the phrasing from Don't to Done
+        if @step == @lastactionstep
```

Comment:
i don't understand how this works.

isn't last action step always company? you're using the class definition

why would active pass descriptions have multiple things?

Reply:
ahhhh never mind you set it on line 335

Reply:
step is :track and the like

Reply:
how about

self.class::ACTIVE_PASS_DESCRIPTION[@step == @last_action_step ? @step : nil] || 
  self.class::INACTIVE_PASS_DESCRIPTION[@step] ||
  'Pass'

Reply:
Ahh, ruby experience shines :)